packages/backend/README.md

```diff
@@ -5,6 +5,20 @@
 ### Prerequisites
 
 - [Docker](https://docs.docker.com/engine/install/) configured to [run as non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)
+- If you are running MacOS on a non apple silicon device there is a known [issue](https://github.com/coilhq/tigerbeetle/issues/92) with
+  [tigerbeetle](https://github.com/coilhq/tigerbeetle). To fix this you can set defaults in `$HOME/.docker/daemon.json` with
+
+  ```
```

Comment:
It's not really an issue with TB as such. It requires what it requires. 
It's more a lacking in the [testcontainers](https://github.com/testcontainers/testcontainers-node)  package

Reply:
There's no API to add linux capabilities or set the ulimits

Reply:
That makes sense. I updated the wording to better reflect the problem. Thanks!

Reply:
@sublimator Does it seem reasonable with the new edits?

Reply:
I'm not really sure @tyleralt. Note that I'm not a rafiki dev

I guess you want to say something like this:
```
TigerBeetle requires the privilege of using memlock functions, usually afforded
by adding the linux capability (IPC_LOCK). Unfortunately, rafiki uses
[testcontainers](https://github.com/testcontainers/testcontainers-node) which
provides no api to configure the containers as so. On MacOS, a workaround is
to ... 
```

To be honest I'm not sure why it seems to work on linux (core Rafiki devs and
Github actions CI). Perhaps linux docker has different defaults ? 

@wilsonianb  

What do you think?